sorachung/pet-journal
# Made add a pet form and page, and pets list page, and pet page

```text
#### Changes Made
1. Modified file `AddPetForm` to include `adding a pet` and functionality.
2.  Modified file `PetsList` to include `showing all pets` and functionality.
3.  Modified file `Pet` to include `showing each pet` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout src-petsList
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > start the app
    > log in as sora
    > go to my pets from the side bar
    > see if three pets show
    > click on one pet's name and see if it takes you to that pet's page with the same info
    > go back to /mypets and click add a pet
    > try adding a pet
    > it should redirect you to /mypets and you should see the new pet added
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
```
